docs/csharp/language-reference/keywords/group-clause.md

```diff
@@ -101,4 +101,4 @@ group person by new {name = person.surname, city = person.city};
  [LINQ Query Expressions](../../../csharp/programming-guide/linq-query-expressions/index.md)   
  [How to: Create a Nested Group](../../../csharp/programming-guide/linq-query-expressions/how-to-create-a-nested-group.md)   
  [How to: Group Query Results](../../../csharp/programming-guide/linq-query-expressions/how-to-group-query-results.md)   
- [How to: Perform a Subquery on a Grouping Operation](../../../csharp/programming-guide/linq-query-expressions/how-to-perform-a-subquery-on-a-grouping-operation.md)
\ No newline at end of file
+ [How to: Perform a Subquery on a Grouping Operation](../../../csharp/programming-guide/linq-query-expressions/how-to-perform-a-subquery-on-a-grouping-operation.md)
```

Comment:
This one has been moved over to ...
```
../../../csharp/linq/perform-a-subquery-on-a-grouping-operation.md
```
Others have moved as well here (i.e., everything in the *linq-query-expressions* folder). I think we should hit these on a future pass, since it seems more efficient to go through the redirection file and nail them all at once in a single PR.